src/classes/AccountModelHealthCheckVMapper.cls

```diff
@@ -65,55 +65,71 @@ public virtual with sharing class AccountModelHealthCheckVMapper {
         AccountModelSettingsModel acctModSettingMod = this.locateAccountModelSettingsService().getAccountModelSettingsModel();
         Map<Id, RecordTypeModel> accountRTModelsById = this.locateRecordTypeService().getModelByIdMapForSObjectType(Account.SObjectType);
 
+        List<HealthCheckGroupVModel> accountHlthCheckGroupVModelsList = new List<HealthCheckGroupVModel>();
+
+        accountHlthCheckGroupVModelsList.add(this.getDefaultAccountModelHealthCheckItems(acctModSettingMod, accountRTModelsById));        
+        accountHlthCheckGroupVModelsList.add(this.getAdministrativeAccountModelHealthCheckItems(acctModSettingMod, accountRTModelsById));        
+        accountHlthCheckGroupVModelsList.add(this.getHouseholdAccountModelHealthCheckItems(acctModSettingMod, accountRTModelsById));        
+
+        return this.combineAccountHealthCheckGroupViewModels(accountHlthCheckGroupVModelsList);
+    }
+
+    /**********************************************************************************
+    * @description Generates a HealthCheckGroupVModel comprised of the combined health 
+    * check results for Default Account, Administrative and Household accounts. 
+    * @param  healthCheckGroupVModelsList A list of HealthCheckGroupVModels
+    * @return A HealthCheckGroupVModel representing the health check results for 
+    * Default, Administrative and Household Accounts.
+    **********************************************************************************/ 
+    @TestVisible
+    private HealthCheckGroupVModel combineAccountHealthCheckGroupViewModels(List<HealthCheckGroupVModel> healthCheckGroupVModelsList){
```

Comment:
This is an elegant solution that made the PR much more readable. It also isolated a unit test scenario.